python/nano/test/pytorch/tests/inference/test_bf16_ipex.py

```diff
@@ -150,6 +151,65 @@ def hello():
         with pytest.raises(AttributeError):
             new_model.width
 
+    def test_bf16_ipex_jit_converter(self):
```

Comment:
You don't have to put two identical unit tests in two different files.
Now `test_bf.py` only contains ut for `bf16` and `bf16+channels_last`, and all `ipex / jit` related ut are put in `test_bf16_ipex.py` and `test_ipex_jit_inference.py`. So just put ut in the last two files.

Reply:
Thank you for the reminding! I am kind of confused by the file names. Thank you! Have updated,